### ISSUE TITLE
probot-stale integration

### ISSUE THREAD

**Issue by audiolion** [microjs/microjs.com#904]
Suggested by @sbrl to manage stale/inactive PRs

https://github.com/probot/stale/

**Comment by tunnckoCore** [comment]
Initially, I was :+1: with that idea, but actually... PRs are just 2 pages and we can manually scan them, also most of them are from last year which isn't too old for me ;d

But yea, good to have for the future, indeed.

**Comment by audiolion** [comment]
Yeah I went through everything in the backlog today so its pretty manageable right now

**Comment by sbrl** [comment]
Maybe we could set it to something like a month for the "waiting message" that asks for updates, and 3 months for closing?

**Comment by audiolion** [comment]
seems reasonable

**Comment by tunnckoCore** [comment]
yep
